### DISCUSSION TITLE
Older windows versions compatibility

### DISCUSSION THREAD

**Discussion by Beatle95** [microsoft/msquic#3204] [Q&A] [answered]
Hello, in [this](https://github.com/microsoft/msquic/discussions/2283#discussioncomment-1954527) answer one of the mainteiners said that you are using some Windows API which didn't exist before Windows 10.
If I have an application which has to be supported on older Windows versions (particularly Windows Server 2012 in my case), shoud I use some other QUIC implementation instead of MsQuic?

Upd: I am using OpenSSL instead of Schannel.

**Comment by nibanks** [comment, marked as answer]
We don't explicitly test older versions anymore, but they _should_ work, so long as you use openssl. Please let me know if you encounter any functional problems (perf will be worse).

**Reply by Beatle95** [reply]
Tested on Windows Server 2012 - everything is good so far.
I think it's worth to be explicitly mentioned in documentation that it **may be used, but not tested on older Windows systems before Win10**, because at this moment this paragraph from [documentation page](https://github.com/microsoft/msquic/blob/main/docs/Platforms.md) looks a bit confusing:

> Optionally, msquic.dll can be built with OpenSSL (see below for more details) instead of Schannel on Windows. This removes the Windows OS dependency on TLS from MsQuic, so MsQuic should work on most Windows 10 based client and server versions.

This shows that it may work on Win10 based systems, but no info about earlier releases.

**Reply by nibanks** [reply]
Feel free to send a PR proposing alternate text. Thanks for trying this out!
